kubeportal/tests/test_api.py

```diff
@@ -60,17 +59,16 @@ def test_api_login(api_client_anon, admin_user):
     # self.assertEqual(cookie['kubeportal-auth']['samesite'], 'Lax,')
     # self.assertEqual(cookie['kubeportal-auth']['httponly'], True)
     data = response.json()
-    assert 5 == len(data)
-    assert 'group_ids' in data
-    assert 'webapp_ids' in data
-    assert 'user_id' in data
+    assert 3 == len(data)
+    assert 'user' in data
     assert 'access_token' in data
-    assert 'k8s_namespace' in data
-    assert data['user_id'] == admin_user.pk
+    assert 'refresh_token' in data
```

Comment:
Why does the login route deliver both, access and refresh token?

Reply:
The access token is short-lived, an when it expires, you can use the refresh token to get a new one:

https://django-rest-framework-simplejwt.readthedocs.io/en/latest/getting_started.html#usage
https://auth0.com/blog/refresh-tokens-what-are-they-and-when-to-use-them/